src/commands/txCommands/util/index.js

```diff
@@ -181,7 +181,9 @@ const readResource$ = (slug, project = PROJECT) =>
 	Rx.Observable.bindNodeCallback(tx.sourceLanguageMethods.bind(tx))(
 		project,
 		slug
-	).retry(5);
+	)
+	.do(null, () => console.log(`error readResource$ ${slug} ${project}`))
+	.retry(5);
```

Comment:
I'm not 100% sure, but this `retry` might need to go before the `.do` since you want it to retry the previous step, not the `do` step

Reply:
the aim was to log every time its retried but you might be right. although it leaves me wondering how the code determines which observable needs to be retried and if it simply disallows logging on each retry